winterfell/webapp/src/js/app.js

```diff
@@ -49,7 +49,7 @@ app.config(function ($routeProvider) {
       templateUrl: "templates/claimView.html"
     })
 
-    .when("/form/:formId", {
+    .when("/form/:claimId/:formId", {
```

Comment:
ah i see what you mean, given the resource is form under a claim, that makes sense to me that that would be the path